src/scss/cdb-components/forms/radio.scss

```diff
@@ -16,14 +25,26 @@
 @import '../../cdb-variables/colors';
 
 .CDB-Radio {
+  width: $baseSize * 2;
```

Comment:
Properties should be ordered position, width, height, opacity, vertical-align, z-index